main.js

```diff
@@ -0,0 +1,125 @@
+const mobileMenuBtn = document.querySelector('.mobile-menu-btn');
+const menuOverlay = document.querySelector('.overlay-wrap');
+const closeMobileMenu = document.querySelector('.cancel-wrap');
+const itemMobileMenu = document.querySelectorAll('.item-menu');
+
+mobileMenuBtn.addEventListener('click', () => {
+  menuOverlay.style.display = 'block';
+});
+
+closeMobileMenu.addEventListener('click', () => {
+  menuOverlay.style.display = 'none';
+});
+
+itemMobileMenu.forEach((element) => {
+  element.addEventListener('click', () => {
+    menuOverlay.style.display = 'none';
+  });
+});
+
+// featured activities section
+
+const activities = document.querySelector('.featured-activities');
+const activitiesArray = [
+  {
+    id: '1',
+    image: 'images/features_01.jpg',
+    tittle: 'Remembering the Children: National Day for Truth & Reconciliation',
+    place: 'ew Iceland Heritage Museum & Independent Artists',
+    city: 'Gimli, MB',
+  },
+
+  {
+    id: '2',
+    image: 'images/features_02.jpg',
+    tittle: "Nuit Blanche Kids Zone at The Children's Museum",
+    place: "Manitoba Children's Museum",
+    city: 'Winnipeg, MB',
+  },
+
+  {
+    id: '3',
+    image: 'images/features_03.jpg',
+    tittle: 'WSO Community Celebration Concert',
+    place: 'Centennial Concert Hall',
+    city: 'Winnipeg, MB',
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure you also generate a **More** button on your mobile version where the page shows **two** speakers by default and other speakers are shown once the button is clicked. Here is the view of the expected button
![Screenshot from 2022-09-14 20-01-00](https://user-images.githubusercontent.com/83042742/190229364-44203269-342b-4455-b95f-53596792c3c3.png)